plugins/sqlfluff-templater-dbt/test/templater_test.py

```diff
@@ -376,59 +313,11 @@ def test__templater_dbt_handle_exceptions(
         )
     finally:
         os.rename(target_fpath, src_fpath)
-        get_adapter(dbt_templater.dbt_config).connections.release()
     assert violations
     # NB: Replace slashes to deal with different platform paths being returned.
     assert violations[0].desc().replace("\\", "/").startswith(exception_msg)
 
 
-@pytest.mark.skipif(
-    DBT_VERSION_TUPLE < (1, 0), reason="mocks a function that's only used in dbt >= 1.0"
-)
-@mock.patch("dbt.adapters.postgres.impl.PostgresAdapter.set_relations_cache")
-def test__templater_dbt_handle_database_connection_failure(
```

Comment:
The templater is no longer managing the database connection. We may need to add some sort of test later, but it seems like a lower priority.

Reply:
Now that you've added back in the connection fail logic (https://github.com/sqlfluff/sqlfluff/pull/3976#discussion_r1050766728), should we re-enable this test to make sure that code is properly covered?

Reply:
Found and fixed an old issue with the test that caused some other tests to fail, added it back. Unsure why this wasn't failing already, because it was definitely a bug. Probably related to database connections being managed differently now.